app/src/main/res/values/strings.xml

```diff
@@ -78,8 +70,8 @@
     <string name="okay_button">Okay</string>
     <string name="ok_button">OK</string>
 
-    <string name="askUserToRate">askUserToRate</string>
     <string name="counter">solved</string>
+
```

Comment:
I should put a comment here explaining that these have to be the same in all languages (see comment in app/src/main/java/atorch/statspuzzles/PuzzleSelection.java)